src/router/routes.js

```diff
@@ -162,6 +162,13 @@ export default [
 	{ path: '/process-browser-auth', component: () => import('@/pages/ProcessBrowserAuth') },
 	{ path: '/protocol', component: () => import('@/pages/Protocol') },
 	{ path: '/register/social', component: () => import('@/pages/LoginAndRegister/RegisterSocial') },
+	{
+		path: '/settings',
+		component: () => import('@/pages/Settings/SettingsPage'),
+		meta: {
+			activeLoginRequired: true,
```

Comment:
The top-level settings page only requires a passive login, so this should be `authenticationRequired`